### PR TITLE
Improve AC0251100NJ actions consistency

### PR DESCRIPTION
**Pull request by arteck**

### CODE REVIEW

**Comment by Koenkk** [comment]
Despite some renaming, what is the added value of this?

**Comment by arteck** [comment]
current implementation does not work

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -1931,7 +1931,60 @@ const converters = {
...
+        convert: (model, msg, publish, options) => {
+            return {click: 'circle_hold'};
+        },
+    },
+    
+    AC0251100NJ_circle: {
+        cid: 'lightingColorCtrl',
+        type: 'cmdMoveToSaturation',
+        convert: (model, msg, publish, options) => {
+            return {click: 'circle_click'};
```
Comment (Koenkk) [inline converters/fromZigbee.js]: For consistency purposes, the above 3 should also be `action`.

**Comment by Koenkk** [comment]
What's not working in that implementation?

**Comment by arteck** [comment]
up down and circle...  don't works for me

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -1931,7 +1931,57 @@ const converters = {
...
+        type: 'cmdMove',
+        convert: (model, msg, publish, options) => {
+            return {action: 'off_hold'};
+        },
+    },
+    AC0251100NJ_release: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMoveToLevelWithOnOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'circle_press'};
```
Comment (Koenkk) [inline converters/fromZigbee.js]: What is the difference with the circle_click here? (do you e.g. have to press the button longer?)

File: devices.js [hunk 3]
```diff
@@ -1204,9 +1204,9 @@ const devices = [
         description: 'Smart+ switch mini',
         supports: 'on/off, brightness',
         fromZigbee: [
-            fz.genOnOff_cmdOn, fz.genOnOff_cmdOff, fz.AC0251100NJ_long_middle,
-            fz.cmd_stop, fz.cmd_move, fz.cmd_move_with_onoff,
-            fz.cmd_move_to_level_with_onoff, fz.generic_batteryvoltage_3000_2500,
+            fz.AC0251100NJ_on, fz.AC0251100NJ_off, fz.AC0251100NJ_on_hold, fz.AC0251100NJ_off_hold,
+            fz.AC0251100NJ_release, fz.AC0251100NJ_circle, fz.AC0251100NJ_circle_release,
+            fz.generic_batteryvoltage_3000_2500, fz.cmd_stop,
```
Comment (Koenkk) [inline devices.js]: I think the fz.cmd_stop should be removed and replaced with a different once (as you did with all the others).

**Comment by Koenkk** [comment]
I see now, if you could answer my 2 other comments I can merge this.

File: converters/fromZigbee.js (hunk 2 above)
Comment (arteck) [reply converters/fromZigbee.js]: you have 3 stats here.. pressed, released, hold

File: devices.js (hunk 3 above)
Comment (arteck) [reply devices.js]: with a different once ??

Comment (Koenkk) [reply devices.js]: https://github.com/Koenkk/zigbee-shepherd-converters/blob/master/converters/fromZigbee.js#L1471, it use the `ictcg1` method, for this, it also should use `cmd_move` etc, but these are removed in this commit. Meaning that this once should also be removed/replaced.

Comment (arteck) [reply devices.js]: ahh .. yes i think we can remove this fz.cmd_stop

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**